asset/swift.go

```diff
@@ -54,3 +54,25 @@ func (s *SwiftFS) RemoveAsset(shasum string) error {
 
 	return nil
 }
+
+func (s *SwiftFS) EmptyThenDeleteContainer(name string) error {
+	sc := s.Connection
+
+	// First emptying the container
+	objs, err := sc.ObjectNames(name, nil)
+	if err != nil {
+		return err
+	}
+
+	_, err = sc.BulkDelete(name, objs)
```

Comment:
`BulkDelete` doesn't work with more 10_000 objects at the same time, and deleting more than 10_000 objects one by one will be very slow!

Reply:
Dead code, removed